.github/contribution/README.md

```diff
@@ -40,7 +40,7 @@
 First, you need to build the project:
 
 ```
-yarn build
+yarn build-ci
```

Comment:
could you remove that? we already building the tokens during the build, so no need in that ;)

Reply:
This is the README in the contribution guide, there is no step yarn build, I thought it's out of date? Am I missing something here?

Reply:
The steps to start with development here https://github.com/kiwicom/orbit/blob/master/.github/contribution/README.md#starting-with-development
should be 
```
yarn build-ci
yarn dev
```
right? or at least that's what I had to do, as there was no target for yarn build